.github/workflows/publish.yml

```diff
@@ -4,7 +4,8 @@ on:
   push:
     branches:
       - develop
-      - main
```

Comment:
don't need `main` anymore b/c those releases should be tagged

Reply:
Woot. You rock!